examples/direct-upload/test/e2e/upload.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  'Image Upload': function (browser) {
+    browser
+      .url('http://localhost:8000')
+      .waitForElementVisible('body', 1000)
+      .waitForElementVisible('input#fileInput', 1000)
+      .pause(1000)
+      .setValue('input#fileInput', require('path').resolve(__dirname + '/testfile.png'))
```

Comment:
Any _reason_ to not be using a [`cute kitten`](https://www.google.pt/search?q=cute+kitten&tbm=isch) here...? 😜